src/components/landing/LoginCard.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import LandingCard from './LandingCard.jsx';
+import TextInput from '../TextInput.jsx';
+import Button from '../Button.jsx';
+
+function LoginCard() {
+  async function login(event) {
+    event.preventDefault();
+
+    // TODO API
+  }
+
+  return (
+    <LandingCard title="Welcome Back">
+      <form onSubmit={login}>
```

Comment:
Consider adding the card-form class to the form element so the button and text input take up the same width. That was how I did it for the group login card but if the register card does not do that then we can go either way in terms of making the style consistent among cards.